packages/ruby/src/install-ruby.ts

```diff
@@ -2,28 +2,48 @@ import execa from 'execa';
 import which from 'which';
 import { join } from 'path';
 import { intersects } from 'semver';
-import { Meta, NodeVersion, debug, NowBuildError } from '@vercel/build-utils';
+import {
+  Meta,
+  debug,
+  NowBuildError,
+  ActiveNodeVersion,
+  DeprecatedNodeVersion,
+  DiscontinuedNodeVersion,
+} from '@vercel/build-utils';
 
-interface RubyVersion extends NodeVersion {
+interface ActiveRubyVersion extends ActiveNodeVersion {
```

Comment:
> Do we want RubyVersion to still be dependent on NodeVersion?

This looks like a signal that these types aren't really node and ruby specific. Instead, the are Major-defined and Minor-defined types. Maybe we should rename them to be more generic.